scripts/build-project.sh

```diff
@@ -2,5 +2,5 @@
 set -euv
 
 sh -c "NODE_ENV=production yarn build"
```

Comment:
Beware this still uses `yarn build` instead of `yarn build:clean`, so it's prone to having trash in the `lib` folder when publishing

Reply:
👍